doc/dev/developer_guide/tests-unit-tests.rst

```diff
@@ -1,11 +1,17 @@
 Testing - unit tests
 ====================
 
-Ceph has two types of tests: unit tests (also called ``make check`` tests) and
-integration tests. Strictly speaking, the ``make check`` tests are not "unit
-tests", but rather tests that can be run easily on a single build machine
-after compiling Ceph from source, whereas integration tests require packages
-and multi-machine clusters to run.
+Ceph has two types of tests: 
+
+#. unit tests (also called ``make check`` tests) and
```

Comment:
why do we an "and" if you are numbering them? https://ceph--38133.org.readthedocs.build/en/38133/dev/developer_guide/tests-unit-tests/#testing-unit-tests

Reply:
It's a good question. In this case, I was trying to keep it read-out-loud-able. However, since it clearly strikes the reader (you) as wrong, I'm going to change it.

Thanks for this, Neha.